cluster-autoscaler/cloudprovider/aws/ec2_instance_types.go

```diff
@@ -1509,32 +1607,25 @@ var InstanceTypes = map[string]*InstanceType{
 		GPU:          0,
 		Architecture: "amd64",
 	},
-	"g3.16xlarge": {
-		InstanceType: "g3.16xlarge",
-		VCPU:         64,
-		MemoryMb:     499712,
-		GPU:          4,
-		Architecture: "amd64",
-	},
-	"g3.4xlarge": {
-		InstanceType: "g3.4xlarge",
-		VCPU:         16,
-		MemoryMb:     124928,
-		GPU:          1,
+	"f2.12xlarge": {
+		InstanceType: "f2.12xlarge",
+		VCPU:         48,
+		MemoryMb:     524288,
+		GPU:          0,
 		Architecture: "amd64",
 	},
-	"g3.8xlarge": {
-		InstanceType: "g3.8xlarge",
-		VCPU:         32,
-		MemoryMb:     249856,
-		GPU:          2,
+	"f2.48xlarge": {
+		InstanceType: "f2.48xlarge",
+		VCPU:         192,
+		MemoryMb:     2097152,
+		GPU:          0,
 		Architecture: "amd64",
 	},
-	"g3s.xlarge": {
-		InstanceType: "g3s.xlarge",
-		VCPU:         4,
-		MemoryMb:     31232,
-		GPU:          1,
```

Comment:
confirmed that all these instances are deprecated and not on the AWS docs anymore 👍🏾